mkt/websites/views.py

```diff
@@ -2,13 +2,23 @@
 
 from rest_framework.generics import ListAPIView
 from rest_framework.permissions import AllowAny
-from rest_framework.serializers import Serializer
 
 from mkt.api.authentication import (RestOAuthAuthentication,
                                     RestSharedSecretAuthentication)
 from mkt.api.base import CORSMixin, MarketplaceView
 from mkt.api.paginator import ESPaginator
 from mkt.websites.indexers import WebsiteIndexer
+from mkt.websites.models import Website
+from mkt.websites.serializers import ESWebsiteSerializer, WebsiteSerializer
+
+
+class WebsiteView(CORSMixin, MarketplaceView, ListAPIView):
+    cors_allowed_methods = ['get']
+    authentication_classes = [RestSharedSecretAuthentication,
+                              RestOAuthAuthentication]
+    permission_classes = [AllowAny]
+    serializer_class = WebsiteSerializer
+    model = Website
```

Comment:
Not hooked up to a URL yet no - for now, it's just to help testing (I'm manually instantiating the view in the tests)